deployments/charts/kuma/values.yaml

```diff
@@ -193,6 +193,10 @@ ingress:
     kubernetes.io/os: linux
     kubernetes.io/arch: amd64
 
+gateway:
+  # -- Whether the control plane is using an image with built-in Gateway support enabled
+  enabled: false
+
```

Comment:
It's not all that clear to me what policy you have in mind here. Are you expecting that people would need to be able to choose to not install gateway CRDs?

| | kumactl | kuma-cp | Behaviour |
| --- | --- | --- | --- |
| Gateway Enabled | Y | Y | kumactl installs CRDs and kuma-cp works with them|
| Gateway Enabled | Y | N | kumactl installs CRDs but kuma-cp pays no attention |
| Gateway Enabled | N | Y | kumactl doesn't install CRDs and kuma-cp fails to watch the GVK (hopefully not fatal) |
| Gateway Enabled | N | N | No gateway support anywhere, it's all cool |

Reply:
Or maybe the behavior of just always installing the CRDs doesn't seem so bad?

Reply:
Ultimately I went through this effort to avoid changing the output of `install control-plane` at all if gateway isn't enabled. Definitely open to changing that

Reply:
Removed the Helm value here, now `gateways` appear in the RBAC resource.

Reply:
The issue I see is that next release CRDs will appear to users which aren't actually supported

Reply:
> The issue I see is that next release CRDs will appear to users which aren't actually supported

Yeh, I think you are right, we should avoid that. It would be surprising to users and we could accidentally cause compatibility issues for ourselves.